src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
> @thomaslevesque so, you're saying "leave it as it is, except maybe rename the horribly-named field"?

Exactly

> I was going to say that our shim does. But it turns out the real one does too! I'm forever learning things from you.

Oh! Actually, I was thinking only about our shim... I lost sight of the fact that it was only a shim :flushed:
But anyway, why not use the shim all the time? Is there any benefit in using the "real" `ConcurrentQueue<T>`?